core/storage/feedback/gae_models.py

```diff
@@ -419,28 +454,22 @@ def create(cls, exploration_id, thread_id, author_id, exploration_version,
         if cls.get_by_id(instance_id):
             raise Exception('There is already a feedback thread with the given '
                             'thread id: %s' % instance_id)
+        state_content = cls._convert_suggestion_html_to_legacy_state_content(
```

Comment:
As a noob question, is the reason we need this because the Feedback storage model is of the old content schema, and we can't change it unless we make a migration system for Feedback?